.circleci/config.yml

```diff
@@ -28,10 +28,11 @@ jobs:
           command: |
             # Upload build dir to cluster.ipfs.io
             hash=$(ipfs-cluster-ctl \
+              --quietest \
```

Comment:
@hsanjuan using `--quietest` on ipfs-cluster-ctl here causes

```console
Incorrect Usage. flag provided but not defined: -quietest
```

did i use it wrong?

Reply:
Sorry ` --quieter, -Q                          Write only final hash to output`